src/MixpanelEventForwarder.js

```diff
@@ -245,8 +284,20 @@ var constructor = function () {
     this.process = processEvent;
     this.setUserAttribute = setUserAttribute;
     this.setUserIdentity = setUserIdentity;
-    this.onUserIdentified = onUserIdentified;
     this.removeUserAttribute = removeUserAttribute;
+
+    this.onLoginComplete = onLoginComplete;
+
+    // For all Identity Requests, we run mixpanel.identify(<device_id)
+    // as per Mixpanel's documentation https://docs.mixpanel.com/docs/tracking-methods/identifying-users
+    // except when a user logs out, where we run mixpanel.reset() to
+    // detach the distinct_id from the device_id
```

Comment:
For clarity, here and elsewhere, we should refer to device_id as the mixpanel device_id because we never use the mParticle device_id.  Additionally, the context of `distinct_id` is unclear unless you read through the docs.  I think we should call out what the distinct_id is and how it works in Mixpanel for future developers.

Reply:
Good call out.